### DISCUSSION TITLE
Where i have to put the commit hash in the pom.xml?

### DISCUSSION THREAD

**Discussion by aprobeni** [discord-jda/JDA#1636] [Questions and Help] [answered]
Hi!
I don't really know how to test the commit
I have to use the latest commit hash from "feature/slash-commands", which is "b821609"
But i don't really know where i have to put it
This is the pom.xml: 
(sorry, i was not able to upload the pom.xml, so i made it .txt)

[pom.xml.txt](https://github.com/DV8FromTheWorld/JDA/files/6564640/pom.xml.txt)

**Comment by MinnDevelopment** [comment, marked as answer]
https://github.com/DV8FromTheWorld/JDA/wiki/10%29-FAQ#how-do-i-test-other-builds

**Reply by aprobeni** [reply]
So, i guess i have to do this right?
<groupId>com.github.DV8FromTheWorld</groupId>
<artifactId>JDA</artifactId>
<version>feature/slash-commands-SNAPSHOT</version>
But i always get this:
Could not find artifact com.github.DV8FromTheWorld:JDA:pom:feature/slash-commands-SNAPSHOT in dv8tion (https://m2.dv8tion.net/releases)

**Reply by aprobeni** [reply]
nevermind, i found it
thank you!
